sound/soc/intel/boards/sof_sdw.c

```diff
@@ -1442,7 +1455,12 @@ static int sof_card_dai_links_create(struct device *dev,
 		cpus[cpu_id].dai_name = cpu_name;
 		init_dai_link(dev, links + link_index, be_id, name, 1, 1,
 				cpus + cpu_id, 1, ssp_components, 1, NULL, NULL);
+		INC_ID(be_id, cpu_id, link_index);
 	}
+	/* Echo reference */
+	init_dai_link(dev, links + link_index, be_id, "Echo-reference", 0, 1,
+			sof_dummy_component, 1, dummy_component, 1, NULL, NULL);
+	num_links++;
```

Comment:
Should this echo reference be optional?
Like extending the `mach_params` with a bool.

Reply:
> Should this echo reference be optional? Like extending the `mach_params` with a bool.

I thought about this. But I think we can always create an echo reference BE dailink even we don't use it. @plbossart What do you think?